Libraries/LibWeb/XML/XMLDocumentBuilder.h

```diff
@@ -41,6 +41,7 @@ class XMLDocumentBuilder final : public XML::Listener {
     Optional<FlyString> namespace_for_name(XML::Name const&);
 
     GC::Ref<DOM::Document> m_document;
+    GC::Ptr<DOM::Node> m_parent_node;
```

Comment:
do we need to visit this?

Reply:
Because nothing else in this file is visited, my assumption was that this object can't live long enough for GC to be an issue. That probably isn't true though.

Reply:
I've made this a `RootVector` instead.